### DISCUSSION TITLE
Is this plugin still to be used?

### DISCUSSION THREAD

**Discussion by romerod** [rabbitmq/rabbitmq-auth-backend-amqp#40] [General]
### Describe the bug

We are considering using this plugin but we are not sure if it's still maintained.
There are sections in the rabbitmq documentation which say that this plugin is a builtin plugin:
https://www.rabbitmq.com/docs/access-control#backends
The link is even pointing to this https://github.com/rabbitmq/rabbitmq-server/tree/v3.13.x/deps/rabbitmq-auth-backend-amqp which doesn't exist.

On other pages the link is pointing to this repository. In [the readme](https://github.com/rabbitmq/rabbitmq-auth-backend-amqp/blob/main/README.md) it states that this plugin can be downloaded from the [community plugins](http://www.rabbitmq.com/community-plugins.html). There it's not listed.

Thank you in advance for your advice

### Reproduction steps

-

### Expected behavior

-

**Comment by michaelklishin** [comment]
This plugin was never widely used and is not a priority for the core team. The docs will be updated shortly.

**Reply by romerod** [reply]
Are you still willing to maintain it? Like create a version for RabbitMQ 13, etc?

**Reply by lukebakken** [reply]
Probably not, unless a paying customer requests it. Please see https://www.rabbitmq.com/contact#paid-support

If someone in the RabbitMQ community wishes to provide pull requests to continue maintenance of this plugin, we would gladly review and accept them.

**Reply by michaelklishin** [reply]
"Not a priority" does not mean that it will be abandoned completely but it's up to those who use this plugin to contribute most of the work (which usually will be primarily metadata updates and small internal API updates). Or a paying customer to request an update.

But all in all, this is a 3rd party plugin now, primarily for the reason of its much lower popularity (compared to the OAuth 2, HTTP service, LDAP authN/authZ backends).
